CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.30.0 - 2020-12-16
+### Added
+- Add run button as inline button in Java Projects Explorer. [PR#900](https://github.com/microsoft/vscode-java-debug/pull/900).
```

Comment:
`Java Projects Explorer` -> `Project Manager for Java`

Reply:
Project Manager for Java is the extension name, Java Projects is the UI name.
![image](https://user-images.githubusercontent.com/14052197/102300146-5393cb00-3f8f-11eb-84cb-fe8a09dab778.png)

@jdneo WDYT?

Reply:
Since we are talking about the explorer, so I think `Java Project` explorer is fine. But please use singular when then noun is used as attributives